docs/installation/advanced_radius_configuration.asciidoc

```diff
@@ -211,6 +211,62 @@ Click on `Save` and restart radiusd service.
 
 Now when a device connect with the username bob@acme.com then the authentication and accounting requests will be forwarded to one of the ACME RADIUS servers.
 
+==== RADIUS Proxy Advanced
+
+In this section we will explain how to proxy a RADIUS request based on advanced criteria.
+
+First you have to create RADIUS authentication source like above and create per example 2 REALMs "to_NPS" and "to_ISE" (Associate the RADIUS sources in the REALMs)
+
+Next you have to enable the RADIUS filters in the packetfence.authorize and packetfence.post-proxy scope, to do that you have to go in _Configuration -> System Configuration -> RADIUS -> General , and enable "Use RADIUS filters in packetfence authorize" and "Use RADIUS filters in packetfence post-proxy".
+
+At this step restart the radius-auth service.
+
+Here some examples of what you can do with the RADIUS filters (the content of the radius_filters.conf file):
+
+Proxy to the to_NPS Realm the RADIUS request if the Calling-Station-Id or Colubris-AVPair attribute match the regex ACME$
+
+  [NPS]
+  scopes=packetfence.authorize
+  description=to_NPS
+  condition=radius_request.Called-Station-Id =~ "ACME$" || radius_request.Colubris-AVPair =~ "ACME$"
+  status=disabled
+  merge_answer=yes
+  answer.0=control:Proxy-To-Realm = to_NPS
+
+Proxy to the to_ISE Realm the RADIUS request if the Calling-Station-Id or Colubris-AVPair attribute contains ACME_Admin$ and add the Realm to_ISE in the RADIUS request (can be per example use as a filter in a connection profile)
```

Comment:
Proxy to the to_ISE Realm the RADIUS request -> Proxy the RADIUS request to the to_ISE realm